packages/commonwealth/client/scripts/views/pages/discussions/CommentCard/CommentCard.tsx

```diff
@@ -1,20 +1,25 @@
+import React, { useState } from 'react';
 import moment from 'moment';
 import type { DeltaStatic } from 'quill';
-import React, { useState } from 'react';
+
 import app from 'state';
-import type Comment from 'models/Comment';
-import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
-import { PopoverMenu } from 'views/components/component_kit/cw_popover/cw_popover_menu';
-import { CWTag } from 'views/components/component_kit/cw_tag';
-import { CWText } from 'views/components/component_kit/cw_text';
-import { CommentReactionButton } from 'views/components/ReactionButton/CommentReactionButton';
-import { ReactQuillEditor } from 'views/components/react_quill_editor';
-import { QuillRenderer } from 'views/components/react_quill_editor/quill_renderer';
-import { deserializeDelta } from 'views/components/react_quill_editor/utils';
-import { SharePopover } from 'views/components/share_popover';
+import useUserActiveAccount from '../../../../hooks/useUserActiveAccount';
 import { AuthorAndPublishInfo } from '../ThreadCard/AuthorAndPublishInfo';
+import type Comment from '../../../../models/Comment';
+import { CWButton } from '../../../components/component_kit/new_designs/cw_button';
+import { PopoverMenu } from '../../../components/component_kit/cw_popover/cw_popover_menu';
+import { CWTag } from '../../../components/component_kit/cw_tag';
+import { CWText } from '../../../components/component_kit/cw_text';
+import { CommentReactionButton } from '../../../components/ReactionButton/CommentReactionButton';
+import { ReactQuillEditor } from '../../../components/react_quill_editor';
+import { QuillRenderer } from '../../../components/react_quill_editor/quill_renderer';
+import { deserializeDelta } from '../../../components/react_quill_editor/utils';
+import { SharePopover } from '../../../components/share_popover';
+import { CWThreadAction } from '../../../components/component_kit/new_designs/cw_thread_action';
+import useForceRerender from '../../../../hooks/useForceRerender';
+import { getCommentSubscriptions, handleToggleSubscription } from '../helpers';
```

Comment:
what is the reason to change all those import paths?

Reply:
I recall reading something on our wiki that would be better to make imports like `'../../views/...'` rather than just doing `'client/scripts/views/...'`

I thought it would be nice to have all of them organized of what they are, like:
```
'../../../../hooks/useUserLoggedIn';
'../../../../state/api/comments';
'../../../components/Comments/CreateComment';
'../../../modals/confirmation_modal';
'../../../../controllers/app/notifications';
```

But no worries, I can change it back if necessary. Thoughts?